pkg/internal/approver/manager/review_test.go

```diff
@@ -41,10 +40,9 @@ func Test_Review(t *testing.T) {
 		cancel()
 	})
 
-	rootDir := env.RootDirOrSkip(t)
 	env := env.RunControlPlane(t, ctx,
-		filepath.Join(rootDir, "_bin/cert-manager"),
-		filepath.Join(rootDir, "deploy/charts/approver-policy/templates/crds"),
+		env.GetenvOrFail(t, "CERT_MANAGER_CRDS"),
+		env.GetenvOrFail(t, "APPROVER_POLICY_CRDS"),
```

Comment:
Maybe use a consistent import alias, `testenv`? Or will that collide with https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.16.3/pkg/envtest? No, I think it will work fine.

Reply:
done